tests/static/__init__.py

```diff
@@ -0,0 +1,83 @@
+import json
+import mock
+import os
+
+from ..settings import settings
+
+
+class MockOutputMixin(object):
+
+    class DisableMockMixin(object):
+        """
+        Disables the mock object in context managers restoring the mocked
+        function to its original state
+        """
+        def assert_called(self, *args, **kwargs):
+            pass
+
+        def assert_not_called(self, *args, **kwargs):
+            pass
+
+        def assert_called_once(self, *args, **kwargs):
+            pass
+
+        def assert_called_with(self, *args, **kwargs):
+            pass
+
+        def assert_called_once_with(self, *args, **kwargs):
+            pass
+
+        def assert_has_calls(self, *args, **kwargs):
+            pass
+
+        def assert_any_call(self, *args, **kwargs):
+            pass
+
+        def start(self):
+            pass
+
+        def stop(self):
+            pass
+
+        def __enter__(self, *args, **kwargs):
+            return self
+
+        def __exit__(self, *args, **kwargs):
+            return True
```

Comment:
The problem with https://github.com/beanbaginc/kgb is lack of a `patch` function like we are using in all the tests for this PR. To use it, it would require changing a lot of code.

Reply:
What better alternative do you propose @purhan? Having the possibility of asserting calls during real tests is a real game changer which we should pursue, only that way we'll be able to run tests with real devices and ensure they're not only returning the right output but we will also be confident our code really works as intended.